### ISSUE TITLE
Netatmo: using the new system of units provided by ESH

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab2-addons#3353]
As soon as the OH snapshot is stabilized, it would be a good idea to integrate in this binding the new system of units provided by ESH.

@clinique : can you explain how are managed the units by Netatmo ? Currently the API is providing the used units as a read-only setting ? I see no setting in the Netatmo app to update any unit.

**Comment by clinique** [comment]
@lolodomo : yes, currently the binding reads this information from the API, and it is read-only. I plan to use system units in the future of the Netatmo binding, but my idea is to leave Netatmo as the master of this setup, so OH would know what units are read, and convert accordingly if needed.

**Comment by lolodomo** [comment]
Ok, perfect.
I can help you if needed.
I take a look how has been updated the WU binding and that should be easy to adapt.
Maybe we should wait for the new system channel types to be defined, and then do only one change.

**Comment by clinique** [comment]
Yes, and I discovered friday that I have huge work on the table due to Netatmo API updates. Many functions are now marked as obsolete. So I plan an update of the swagger definition file, to port them to the binding, and then update it...once again.

**Comment by lolodomo** [comment]
@clinique : finally I think we should not wait for the new system channel types to move to the new unit system.
The current discussion on new system channels makes me think this would very probably lead to only a very short list of system channels for the weather data and it could take ages to decide what should be or not defined as a system channel. By the way, we could switch to these few new system channels the day something is decided.
Until that happens, we can start updating our existing binding channel types to use QuantityType. I can do it. I think this is osmething fully independ on your update of the swagger definition to be done ?

**Comment by clinique** [comment]
@lolodomo : as far as I think of it, it is independant, so you can proceed. I'm a bit stalled on the swagger revamp. Not much time available currently.
